entry_types/scrolled/package/src/editor/vendor/jquery.imgareaselect.module.css

```diff
@@ -0,0 +1,17 @@
+:global .imgareaselect-border1,
+:global .imgareaselect-border2,
+:global .imgareaselect-border3,
+:global .imgareaselect-border4 {
+  opacity: 0.5;
+}
+
+:global .imgareaselect-handle {
+  background-color: #fff;
+  border: solid 1px #000;
+  opacity: 0.5;
+}
+
+:global .imgareaselect-outer {
+  background-color: #000;
+  opacity: 0.8;
```

Comment:
I think the outer area is currently too dark. Especially in blank slate the image is hardly visible. Here an opacity of 0.5 would suffice imo.

Reply:
Changed.